infra/charts/management-service/templates/_helpers.tpl

```diff
@@ -39,7 +39,7 @@ Common labels
 */}}
 {{- define "xp.labels" -}}
 helm.sh/chart: {{ include "xp.chart" . }}
-{{- with .Values.xpManagement.extraLabels }}
+{{- with (.Values.xpManagement).extraLabels }}
```

Comment:
Has this been tested? I'm not sure if we need it to be `((.Values.xpManagement).extraLabels)` (additional paranthesis) instead.

Reply:
Yep, tested this via Turing's helm chart. No need for the additional parenthesis.